### DISCUSSION TITLE
Cannot access AIO nextcloud interface anymore.

### DISCUSSION THREAD

**Discussion by adder** [nextcloud/all-in-one#4369] [Questions] [answered]
### Steps to reproduce
1. Go to the admin overview page
2. click on the "open nextcloud AIO interface" button on the top.

### Expected behavior 
open nextcloud AIO interface

### Actual behavior 
```
Web page with follow error:

Warning: Undefined array key "AIO_TOKEN" in /var/www/docker-aio/php/src/Data/ConfigurationManager.php on line 26
Slim Application Error

A website error has occurred. Sorry for the temporary inconvenience.
[Go Back](https://localhost:8080/api/auth/getlogin?token=f285a1dd2caff0a3e1d6a20cade46362ec77a295fc685ee9#)
```
![image](https://github.com/nextcloud/all-in-one/assets/706666/aedb5751-b66c-4b58-9d2a-9a51e849620a)

### Host OS 
Arch linux

#### Nextcloud AIO version 
[Nextcloud Hub 6](https://nextcloud.com/) (27.1.5)

#### Current channel 
stable? (unknown)

**Comment by szaimen** [comment, marked as answer]
Hi, I fear I cannot reproduce this. For debugging, can you post the output of `sudo docker exec nextcloud-aio-mastercontainer cat /mnt/docker-aio-config/data/configuration.json` here? (but please mask all sensible data)
Also, can you try if the login works if you first run `sudo docker stop nextcloud-aio-apache` and then visit https://localhost:8080 and log in using your mastercontainer password?
